client/components/OperationList.js

```diff
@@ -165,6 +165,142 @@ function ComputeAttachmentLink(op) {
     return `operations/${op.id}/`+file;
 }
 
+
+
+class LabelComponent extends React.Component {
+    constructor(props) {
+        super(props);
```

Comment:
Can you add a few prerequisites on `props`? Like `has(props, 'operation'); has(props, 'defaultValue');`, etc.